packages/eslint-plugin-react-hooks/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "eslint-plugin-react-hooks",
   "description": "ESLint rules for React Hooks",
-  "version": "4.1.2",
+  "version": "4.2.0",
```

Comment:
What was the point of this `eslint-plugin-react-hooks` minor version bump, it appears there are no changes? Also, this new version was published without a changelog entry:

https://github.com/facebook/react/blob/master/packages/eslint-plugin-react-hooks/CHANGELOG.md

Reply:
No particular reason other than our release script releases all packages.

Reply:
Edited changelog.

Reply:
Such a script is undesirable. Millions of developers are needlessly updating a "outdated" dependency, looking around for a changelog entry (that doesn't exist) to explain the new semver minor features (that also don't exist).

Reply:
I added the changelog entry. Your opinion is noted. This was a stressful release that took us many months, and I overlooked this detail. I'm sorry to hear you were frustrated by it.